<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>950</width>
    <height>760</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>房贷计算器1</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Form{
border-image: url(:/C:/Users/dell/Desktop/QQ图片20200527165520.jpg);
}</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>80</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;货款类别：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>70</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="editable">
    <bool>false</bool>
   </property>
   <property name="currentText">
    <string>商业贷款</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>商业贷款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>公积金贷款</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>组合型贷款</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>160</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;商业贷款额：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;商贷利率：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>320</y>
     <width>165</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;公积金贷款额：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>400</y>
     <width>141</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;公积金利率：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>480</y>
     <width>118</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;按揭年数：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>620</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 18pt &quot;Arial Black&quot;;
background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>开始计算</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>620</y>
     <width>201</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 18pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>清空重填</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>150</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入商业贷款额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>230</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入商贷利率</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>310</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入公积金贷款额</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>390</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>请输入公积金利率</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>470</y>
     <width>591</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>25年（300期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>24年（288期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>23年（276期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>22年（264期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>21年（252期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>20年（240期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>19年（228期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>18年（216期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>17年（204期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>16年（192期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>15年（180期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>14年（168期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>13年（156期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>12年（144期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>11年（132期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>10年（120期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>9年（108期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>8年（96期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>7年（84期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>6年（72期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>5年（60期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>4年（48期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>3年（36期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>2年（24期）</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>1年（12期）</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>160</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 14pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>万</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>310</y>
     <width>68</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;万&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>230</y>
     <width>68</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>400</y>
     <width>68</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>401</width>
     <height>39</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;还款方式：&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="styleSheet">
       <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>等额本金</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="radioButton">
      <property name="styleSheet">
       <string notr="true">font: 87 14pt &quot;Arial Black&quot;;
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>等额本息</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>35</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>300</x>
     <y>338</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>375</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>308</x>
     <y>423</y>
    </hint>
    <hint type="destinationlabel">
     <x>40</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>textChanged(QString)</signal>
   <receiver>Form</receiver>
   <slot>lineeditchanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>260</x>
     <y>262</y>
    </hint>
    <hint type="destinationlabel">
     <x>25</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>qingkong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>654</y>
    </hint>
    <hint type="destinationlabel">
     <x>839</x>
     <y>646</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Form</receiver>
   <slot>jisuan()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>642</y>
    </hint>
    <hint type="destinationlabel">
     <x>127</x>
     <y>671</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>lineeditchanged()</slot>
  <slot>qingkong()</slot>
  <slot>jisuan()</slot>
 </slots>
</ui>
